### PR TITLE
Added "IgnoreAttribute" configuration to ignore the attributes and only check node names

### PR DESCRIPTION
**Pull request by vfportero**
In my project i need to compare two xml files to check that files have the same nodes (in any order).
With the previous version of XmlUnit i couldn't check this. 

To solve this i added a new property in "DiffConfiguration.cs" call "IgnoreAttribute" (false by default).
In "XmlDiff.cs" the method "CompareAttributes" is ignored if the flag "IgnoreAttribute" is true.

### CODE REVIEW

File: XmlUnit.xUnit/DiffConfiguration.cs [hunk 1]
```diff
@@ -1,92 +1,98 @@
...
+        {
+            get { return _whitespaceHandling; }
+        }
+
+        public bool IgnoreAttributeOrder
+        {
+            get { return ignoreAttributeOrder; }
+        }
+
+        public bool IgnoreAttribute { get; set; }
```
Comment (vfportero) [inline XmlUnit.xUnit/DiffConfiguration.cs:96]: added new property "IgnoreAttribute"

File: XmlUnit.xUnit/XmlDiff.cs [hunk 2]
```diff
@@ -1,466 +1,469 @@
...
+                }
+                CompareAttributes(result, controlAttributes, testAttributes);
+            }
+        }
+
+        private void CompareAttributes(DiffResult result,
+                                       XmlAttribute[] controlAttributes,
+                                       XmlAttribute[] testAttributes)
+        {
+            if (!_diffConfiguration.IgnoreAttribute)
```
Comment (vfportero) [inline XmlUnit.xUnit/XmlDiff.cs:211]: This method are ignored if IgnoreAttribute flag is true

**Comment by horsdal** [comment]
Hey thanks. 
Seems cool. 
Could you add a few unit tests to XmlUnit.Xunit / XmlUnit.xUnitTests for this, please?

**Comment by vfportero** [comment]
Hi Christian!

I added two test to check the default value of "IgnoreAttributes" (false by default) and a test to check that those xml are equal with the flag set as true:

Note: Use '[]' instead ltg markers

[elem value='A']text[/elem]
[elem value='B']text[/elem]

I've tested in Production Environment and works as I expected :)

**Pull request merged**